src/components/header/renderer.tsx

```diff
@@ -125,16 +128,27 @@ class Header extends React.PureComponent<Props, State> {
     } else return null;
   }
 
-  public handleHelpModalToggle(Toggleevent, openPortal, closePortal, isOpen) {
+  public handleAppShortcuts() {
     window.addEventListener('keydown', event => {
       if (
         (event.keyCode === KEYCODES.SINGLE_QUOTE && event.metaKey && !event.shiftKey) || // Handle key press in Mac
         (event.keyCode === KEYCODES.SLASH && event.ctrlKey && event.shiftKey) // Handle Key press in PC
       ) {
-        if (!isOpen) {
-          openPortal();
+        const modalInstance: any = this.refs.helpModal;
+        if (!modalInstance.isOpen) {
+          modalInstance.openPortal();
         } else {
-          closePortal();
+          modalInstance.closePortal();
+        }
+      }
+      // shortcut for opening examples modal
+      if (
+        (event.keyCode === KEYCODES.THREE && event.metaKey && !event.shiftKey) || // Handle key press in Mac
+        (event.keyCode === KEYCODES.THREE && event.ctrlKey && event.shiftKey) // Handle Key press in PC
+      ) {
+        const modalInstance: any = this.refs.examplesModal;
```

Comment:
Also avoid any here.

Reply:
cool, I will use callback refs instead. Does it sound good ?

Reply:
Sure. Do you see the deprecation warning?